datasets/cn/peoples_congress_wikipedia/crawler.py

```diff
@@ -71,7 +104,8 @@ def crawl_item(
         entity,
         position,
         False,
-        start_date="2023",
+        start_date=date_of_by_election or "2023",
+        end_date=date_of_resignation or date_of_dismissal or date_of_death,
         categorisation=categorisation,
         status=OccupancyStatus.UNKNOWN,
```

Comment:
this overrides the status. So to get ENDED, only set status to non-None if you don't have an end date.